### PR TITLE
Skip more sanity checks for secondary create queries

### PR DESCRIPTION
**Pull request by tavplubix**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Follow-up to https://github.com/ClickHouse/ClickHouse/pull/60159

### CODE REVIEW

**Comment by robot-ch-test-poll4** [comment]
*This is an automated comment for commit ace983f3d5d20c4cfaf6220cda7e3251bfb93278 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/61799/ace983f3d5d20c4cfaf6220cda7e3251bfb93278/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: src/Databases/DatabaseOnDisk.cpp [hunk 1]
```diff
@@ -75,7 +75,7 @@ std::pair<String, StoragePtr> createTableFromAST(
         auto table_function = factory.get(table_function_ast, context);
         ColumnsDescription columns;
         if (ast_create_query.columns_list && ast_create_query.columns_list->columns)
-            columns = InterpreterCreateQuery::getColumnsDescription(*ast_create_query.columns_list->columns, context, true, false);
+            columns = InterpreterCreateQuery::getColumnsDescription(*ast_create_query.columns_list->columns, context, LoadingStrictnessLevel::ATTACH);
```
Comment (SmitaRKulkarni) [inline src/Databases/DatabaseOnDisk.cpp:78]: Any specific reason we want to pass `ATTACH` ? I mean why not pass the `mode` passed to `createTableFromAST` function

**Review by SmitaRKulkarni — APPROVED** [review]
Rest all LGTM

File: src/Databases/DatabaseOnDisk.cpp (hunk 1 above)
Comment (tavplubix) [inline src/Databases/DatabaseOnDisk.cpp:78]:
```suggestion
            columns = InterpreterCreateQuery::getColumnsDescription(*ast_create_query.columns_list->columns, context, mode);
```

**Comment by tavplubix** [comment]
CI is broken after enabling analyzer

**Pull request merged**